test failure maybe real? 
```
======================================================================
FAIL: custom.enikshay.ucr.tests.test_episode_datasource:TestEpisodeDatasourceES.test_diagnostic_test_after_end_of_ip
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/ucr/tests/test_episode_datasource.py", line 206, in test_diagnostic_test_after_end_of_ip
    self.assertEqual(query.count(), 1)
AssertionError: 0 != 1
----------------------------------------------------------------------
Ran 2383 tests in 907.636s

```

--- a/custom/enikshay/ucr/data_sources/episode.json
+++ b/custom/enikshay/ucr/data_sources/episode.json
@@ -471,7 +471,6 @@
             {
                 "display_name": "episode_type_patient",
                 "type": "boolean",
-                "create_index": true,
                 "filter": {
                     "type": "named",
                     "name": "current_episode_type_patient"

